### PR TITLE
Brett Schwartz Open Source Proposals

### PR DESCRIPTION
**Pull request by bschwartz10**

### CODE REVIEW

File: 1701-b/4module/open_source_proposals/brett_schwartz.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+### Pitch 1: Put counts of each faker element in the documentation
 
-* **Why:** 
-* **What:** 
-* **I Need:** 
-* **When I'm Done:** 
+* **Why:** I use faker alot in my projects and think it would be beneficial to know how many records were in each faker element.
+* **What:** Put counts of each faker element in the documentation
+* **I Need:** I need to figure out the most efficient solution to take the count of each faker element
+* **When I'm Done:** The issue was just posted and I reached out the the creator of the issue to talk about the best solution. When I'm done, the Faker docs will contain the count of each element.
```
Comment (case-eee) [inline 1701-b/4module/open_source_proposals/brett_schwartz.md]: love that you reached out to the creator of the issue already 👍

File: 1701-b/4module/open_source_proposals/brett_schwartz.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-### Pitch 2: 
+### Pitch 2: Add unit test to Reentry
 
-* **Why:** 
-* **What:** 
-* **I Need:** 
-* **When I'm Done:** 
+* **Why:** Love the cause and the app is looking for open source contribs
+* **What:** Add unit tests for models. Currently only some of the models are being tested at the unit level. 
+* **I Need:** They are using minitest so I have to dive into using minitest on a Rails project.
```
Comment (case-eee) [inline 1701-b/4module/open_source_proposals/brett_schwartz.md]: let us know if you need help with this!

File: 1701-b/4module/open_source_proposals/brett_schwartz.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+* **When I'm Done:** Models will have more test coverage.
 
 ### Pitch 3: 
 
-* **Why:** 
-* **What:** 
-* **I Need:** 
-* **When I'm Done:** 
+* **Why:** Great cause, looking for devs to help out with test coverage
```
Comment (case-eee) [inline 1701-b/4module/open_source_proposals/brett_schwartz.md]: what is this cause? i don't see what project this pitch is for!

Comment (bschwartz10) [inline 1701-b/4module/open_source_proposals/brett_schwartz.md]: Sorry I forgot to include the project. The name is Reentry Double H Ranch

**Pull request merged**